### DISCUSSION TITLE
Query regarding grain size effect

### DISCUSSION THREAD

**Discussion by sushdash** [neperfepx/FEPX#61] [Q&A] [answered]
Hii! 
Just one small query. Is Hall-Petch concept incorporated in this software package? I am trying to deform gradient microstructures, so will FEpX be efficient in manifesting the desired mechanical behavior?

**Comment by rquey** [comment, marked as answer]
No, there is no inherent length scale or grain size effect in FEPX (see https://github.com/acmelab-ua/FEPX/discussions/47#discussioncomment-1545028).

**Comment by sushdash** [comment]
Hii!
Thank you for responding.
Actually, I want to incorporate Hall-Petch into the code, so can you please guide me through the same? Like, where should I make the changes?

**Reply by mkasemer** [reply]
Hi @sushdash - are you interested in submitting a contribution to the code? Please email me at mkasemer@eng.ua.edu and I will discuss where code changes need to take place to institute Hall-Petch style strengthening to FEPX.

**Reply by sushdash** [reply]
Hii!
Yeah, I will be mailing you the required information.
